python/src/pdf417decoder/Decoder.py

```diff
@@ -1167,8 +1167,8 @@ def convert_image(self) -> bool:
         black_white = cv2.threshold(gray, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU)[1]
         
         # padding with single white line at the leftmost of the image
-#        if (sum(black_white[:, 0]) / 255) / black_white.shape[0] < 1:
-#            black_white = np.concatenate([np.full((black_white.shape[0],1), fill_value=255, dtype=black_white.dtype), black_white], axis=1)
+        if (sum(black_white[:, 0]) / 255) / black_white.shape[0] < 1:
```

Comment:
I'm sorry, looks like the previous code was commented out, please use this version of code instead to fix the issue where barcode wasn't able to be detected.